package.json

```diff
@@ -123,7 +125,7 @@
     "chalk": "^2.4.2",
     "concurrently": "^5.1.0",
     "cpx": "^1.5.0",
-    "cross-env": "5.2.0",
+    "cross-env": "^7.0.3",
```

Comment:
Big version change

Reply:
yeah, I'm actually not sure what happened here.  Some kind of merge conflict meant it got uninstalled, so I just re-installed at latest.

It's job is pretty simple: just setting env vars on startup, so... *shrugs*.